cdragontoolbox/export.py

```diff
@@ -659,3 +661,28 @@ def convert(self, fin, output, path):
                 name = os.path.join(obj_output_path, entry["name"] + ".obj")
                 with open(name, "w") as f:
                     f.write(sknfile.to_obj(entry))
+
+class RstConverter(FileConverter):
+    def __init__(self, regex):
+        self.regex = regex
+        self.hashes = hashfile_rst.load()
+
+    def is_handled(self, path):
+        return self.regex.search(path) is not None
+
+    def converted_paths(self, path):
+        yield path + '.json'
+
+    def convert(self, fin, output, path):
+        output_path = os.path.join(output, path + '.json')
+        rstfile = RstFile(fin)
+        hashes = {key_to_rsthash(hash, rstfile.hash_bits): value for hash, value in self.hashes.items()}
+        rst_json = {"entries": {}}
+        for key, value in rstfile.entries.items():
+            if key in hashes:
+                key = hashes[key]
+            rst_json["entries"][key] = value
```

Comment:
I overlooked this at first, but you should use the same format as usual for unresolved hashes:
```py
if key in hashes:
    key = hashes[key]
else:
    key = f"{{{key:10x}}}"
```
Note: I used `10x` assuming 10 hex digit max.